log-appender/src/main/java/io/phdata/pulse/log/HttpAppender.java

```diff
@@ -73,9 +73,7 @@ protected void append(LoggingEvent event) {
         event.setProperty(HOSTNAME, hostname);
       }
 
-      if (shouldFlush(event)) {
-        flush();
-      }
+      flush();
```

Comment:
Flushing the buffer now happens for every logging event?  Also, if you remove `shouldFlush()` from this method, is it used elsewhere in the class? Or should it be removed too?

Reply:
my master branch was borked, opening a new pull request